### ISSUE TITLE
Error on https://www.comdotgame.com/play/kasumi-rebirth

### ISSUE THREAD

**Issue by Hevekwbeie** [ruffle-rs/ruffle#10042]
# Error Info
Error name: Error
Error message: panicked at 'called `Option::unwrap()` on a `None` value', core/src/display_object/container.rs:640:18
Error stack:
```
@https://echo.comdotgame.com/ruffle/core.ruffle.77b76ff8ebf10abc4963.js:1:48787
wasm-stub@[wasm code]
<?>.wasm-function[core::ops::function::FnOnce::call_once{{vtable.shim}}::h321a9078ff9936f2]@[wasm code]
<?>.wasm-function[std::panicking::rust_panic_with_hook::h8d0a63d0479a36c8]@[wasm code]
<?>.wasm-function[core::panicking::panic_fmt::h2a7f454a5f5da218]@[wasm code]
<?>.wasm-function[core::panicking::panic::ha3ebcac1438d9280]@[wasm code]
<?>.wasm-function[ruffle_core::display_object::container::ChildContainer::replace_at_depth::h2de1d6a0c6867e00]@[wasm code]
<?>.wasm-function[ruffle_core::display_object::container::TDisplayObjectContainer::replace_at_depth::h21501d2f3172dfbe]@[wasm code]
<?>.wasm-function[ruffle_core::display_object::movie_clip::MovieClip::instantiate_child::h195a33510c0b747d]@[wasm code]
<?>.wasm-function[ruffle_core::display_object::movie_clip::MovieClip::run_goto::{{closure}}::h7bf1329a46718fc5]@[wasm code]
<?>.wasm-function[ruffle_core::display_object::movie_clip::MovieClip::run_goto::hff866e088751f047]@[wasm code]
<?>.wasm-function[ruffle_core::display_object::movie_clip::MovieClip::goto_frame::h13f7ed5f8308127d]@[wasm code]
<?>.wasm-function[ruffle_core::avm1::globals::movie_clip::goto_frame::ha03b3597855b7782]@[wasm code]
<?>.wasm-function[core::ops::function::FnOnce::call_once::h1fd9f65df876b81f]@[wasm code]
<?>.wasm-function[ruffle_core::avm1::function::Executable::exec::h38c958325b439c56]@[wasm code]
<?>.wasm-function[<ruffle_core::avm1::object::Object as ruffle_core::avm1::object::TObject>::call_method::h7abbd9838b46d90c]@[wasm code]
<?>.wasm-function[ruffle_core::avm1::activation::Activation::run_actions::h9017a8ee32124b3e]@[wasm code]
<?>.wasm-function[ruffle_core::avm1::function::Executable::exec::h38c958325b439c56]@[wasm code]
<?>.wasm-function[<ruffle_core::avm1::object::Object as ruffle_core::avm1::object::TObject>::call::h1b6e751028f92ff8]@[wasm code]
<?>.wasm-function[ruffle_core::avm1::activation::Activation::run_actions::h9017a8ee32124b3e]@[wasm code]
<?>.wasm-function[ruffle_core::avm1::function::Executable::exec::h38c958325b439c56]@[wasm code]
<?>.wasm-function[<ruffle_core::avm1::object::Object as ruffle_core::avm1::object::TObject>::call_method::h7abbd9838b46d90c]@[wasm code]
<?>.wasm-function[ruffle_core::avm1::activation::Activation::run_actions::h9017a8ee32124b3e]@[wasm code]
<?>.wasm-function[ruffle_core::avm1::runtime::Avm1::run_stack_frame_for_action::h1339aa78b7577250]@[wasm code]
<?>.wasm-function[ruffle_core::player::Player::run_actions::hd36ae2ead33b20c6]@[wasm code]
<?>.wasm-function[ruffle_web::Ruffle::tick::h2000682d298368c3]@[wasm code]
<?>.wasm-function[ruffle_web::Ruffle::new_internal::{{closure}}::{{closure}}::{{closure}}::h820d06e21ce20de4]@[wasm code]
<?>.wasm-function[<dyn core::ops::function::FnMut<(A,)>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::ha6dba4b7bc395bcc]@[wasm code]
wasm-stub@[native code]
E@https://echo.comdotgame.com/ruffle/core.ruffle.77b76ff8ebf10abc4963.js:1:3044
f@https://echo.comdotgame.com/ruffle/core.ruffle.77b76ff8ebf10abc4963.js:1:2496
```

# Player Info
Allows script access: true
Player type: Object
SWF URL: https://echo.comdotgame.com/4x90xhl3k0/games/files/f7313836a835b6c3191ed00ca9751a1c.swf
Param allowscriptaccess: always
Param bgcolor: ffffff
Param wmode: gpu
Param base: https://echo.comdotgame.com/4x90xhl3k0/games/files/kasumi-rebirth/3.30/
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined
Attribute 4: undefined
Attribute 5: undefined
Attribute 6: undefined

# Page Info
Page URL: https://www.comdotgame.com/play/kasumi-rebirth
SWF URL: https://echo.comdotgame.com/4x90xhl3k0/games/files/f7313836a835b6c3191ed00ca9751a1c.swf

# Browser Info
User Agent: Mozilla/5.0 (iPhone; CPU iPhone OS 16_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.2 Mobile/15E148 Safari/604.1
Platform: iPhone
Has touch support: true

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-03-12
Channel: nightly
Built: 2023-03-12T00:19:04.440Z
Commit: 1deebc87e869387c962f97b665d26785d0a2b52f
Is extension: false

# Metadata
width: 800
height: 600
frameRate: 24
numFrames: 1
swfVersion: 8
backgroundColor: #000000
isActionScript3: false
uncompressedLength: 3848978

**Comment by josueNHR** [comment]
What does the crash message say?
panicked at 'called Option::unwrap() on a None value', core\src\display_object\container.rs:644:18

Device information
Ruffle Version
0.1.0-nightly (https://github.com/ruffle-rs/ruffle/commit/32f8330199d6c244e41d15fcbb9a9d028fc83af8 2023-03-16)

Affected platform
Desktop app

Operating system
Windows 10.0.22621 (Windows 11 CoreSingleLanguage) [64-bit]

Additional information
Filename: FFHC.swf

Render Info
Renderer: wgpu
Adapter Backend: Vulkan
Adapter Name: "Intel(R) Iris(R) Xe Graphics"
Adapter Device Type: IntegratedGpu
Adapter Driver Name: "Intel Corporation"
Adapter Driver Info: "Intel driver"
Enabled features: PUSH_CONSTANTS | TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES
Available features: DEPTH_CLIP_CONTROL | DEPTH32FLOAT_STENCIL8 | TEXTURE_COMPRESSION_BC | TEXTURE_COMPRESSION_ETC2 | TEXTURE_COMPRESSION_ASTC_LDR | INDIRECT_FIRST_INSTANCE | TIMESTAMP_QUERY | PIPELINE_STATISTICS_QUERY | SHADER_FLOAT16 | MAPPABLE_PRIMARY_BUFFERS | TEXTURE_BINDING_ARRAY | BUFFER_BINDING_ARRAY | STORAGE_RESOURCE_BINDING_ARRAY | SAMPLED_TEXTURE_AND_STORAGE_BUFFER_ARRAY_NON_UNIFORM_INDEXING | UNIFORM_BUFFER_AND_STORAGE_TEXTURE_ARRAY_NON_UNIFORM_INDEXING | MULTI_DRAW_INDIRECT | MULTI_DRAW_INDIRECT_COUNT | ADDRESS_MODE_CLAMP_TO_BORDER | POLYGON_MODE_LINE | POLYGON_MODE_POINT | CONSERVATIVE_RASTERIZATION | VERTEX_WRITABLE_STORAGE | CLEAR_TEXTURE | SPIRV_SHADER_PASSTHROUGH | SHADER_PRIMITIVE_INDEX | MULTIVIEW | TEXTURE_FORMAT_16BIT_NORM | ADDRESS_MODE_CLAMP_TO_ZERO | WRITE_TIMESTAMP_INSIDE_PASSES
Current limits: Limits { max_texture_dimension_1d: 16384, max_texture_dimension_2d: 16384, max_texture_dimension_3d: 2048, max_texture_array_layers: 256, max_bind_groups: 4, max_bindings_per_bind_group: 640, max_dynamic_uniform_buffers_per_pipeline_layout: 8, max_dynamic_storage_buffers_per_pipeline_layout: 0, max_sampled_textures_per_shader_stage: 16, max_samplers_per_shader_stage: 16, max_storage_buffers_per_shader_stage: 0, max_storage_textures_per_shader_stage: 0, max_uniform_buffers_per_shader_stage: 11, max_uniform_buffer_binding_size: 16384, max_storage_buffer_binding_size: 0, max_vertex_buffers: 8, max_buffer_size: 268435456, max_vertex_attributes: 16, max_vertex_buffer_array_stride: 255, min_uniform_buffer_offset_alignment: 64, min_storage_buffer_offset_alignment: 64, max_inter_stage_shader_components: 60, max_compute_workgroup_storage_size: 0, max_compute_invocations_per_workgroup: 0, max_compute_workgroup_size_x: 0, max_compute_workgroup_size_y: 0, max_compute_workgroup_size_z: 0, max_compute_workgroups_per_dimension: 0, max_push_constant_size: 96 }
Surface quality: low
Surface samples: 1
Surface size: Extent3d { width: 800, height: 600, depth_or_array_layers: 1 }

**Comment by n0samu** [comment]
This should be fixed by #10362
